test_classifier.py

```diff
@@ -9,28 +9,26 @@
 #                     NPI Project                     #
 #                        2020                         #
 
+import argparse
 import pickle as pkl
-import torch
-import numpy as np
-from matplotlib import pyplot as plt
 
-from train_classifier import Classifier, extract_needed_layers
```

Comment:
I actually found it necessary to import the Classifier class in order to load Classifier objects correctly via torch :)